Projects/hCoreUI/Sources/CheckboxPickerScreen.swift

```diff
@@ -14,21 +27,24 @@ public struct CheckboxPickerScreen<T>: View where T: Equatable & Hashable {
     private let manualInputPlaceholder: String
     private let hButtonText: String
     private let infoCard: CheckboxInfoCard?
+    private var title: String?
+    private var subTitle: String?
 
     @State var type: CheckboxFieldType? = nil
 
     @State private var selectedItems: [T] = []
     @Environment(\.hButtonIsLoading) var isLoading
     @Environment(\.hCheckboxPickerBottomAttachedView) var bottomAttachedView
     @Environment(\.hIncludeManualInput) var includeManualInput
+    @Environment(\.hUseColoredCheckbox) var coloredCheckBox
 
     @State var manualBrandName: String = ""
     @State var manualInput: Bool = false
 
     private var fieldSize: hFieldSize
```

Comment:
can we change this to use     
@Environment(\.hFieldSize) var size?

and instead of passing it in init, we can use it 
by  .hFieldSize(.small || .large) after we init CheckboxPickerScreen

Reply:
I'm not sure this will work since we want to initialize with

```
            if items.count > 3 {
                self.fieldSize = .small
            } else {
                self.fieldSize = .large
            }
```
if we don't get any value on size. One option could be to check this from where we call CheckboxPickerScreen instead but I don't know if that's better than the current solution?

Reply:
Yeah, ok. Thats fine then :)

Reply:
Yeah, we could do this in the future though if we don't need this check anymore :)